build.xml

```diff
@@ -57,4 +64,16 @@
       <arg line="${example.args}"/>
     </java>
   </target>
+
+  <target name="send_feed" depends="build"
+      description="Send feed example">
+    <property name="example.feed" value="/full.xml"/>
```

Comment:
feed becomes feedFile, and gsa becomes gsaHost, and both expanded names are a little more explicable. I'd prefer example.feedfile and example.gsahost.

For Topaz, we included example.args = "${example.feed} ${example.gsa}" to make it easier to override some or all of the command line. Just a suggestion.